versioned_docs/version-5/examples.mdx

```diff
@@ -1,7 +1,16 @@
 ---
-title: 'Request examples'
+title: XState examples
 ---
 
-**Examples are coming soon**. If you have any examples you want us to make, please [add a request to our feedback board](https://feedback.stately.ai/examples) or upvote an existing suggestion. 
+XState v5 examples are available [in the `/examples` directory](https://github.com/statelyai/xstate/tree/next/examples), or on CodeSandbox:
+
+- [Simple fetch example](https://codesandbox.io/p/sandbox/github/statelyai/xstate/tree/next/examples/fetch)
+- [7GUIs counter (React)](https://codesandbox.io/p/sandbox/github/statelyai/xstate/tree/next/examples/react-7guis-counter)
+- [7GUIs temperature (React)](https://github.com/statelyai/xstate/tree/next/examples/react-7guis-temperature)
+- [Simple list (React)](https://codesandbox.io/p/sandbox/github/statelyai/xstate/tree/next/examples/react-list)
+- [Tic-tac-toe game (React)](https://codesandbox.io/p/sandbox/github/statelyai/xstate/tree/next/examples/react-tic-tac-toe)
```

Comment:
Tic Tac Toe sandbox is using v4 and `createModel()` Is there another link for v5?